data-structures/arrays/min_num_of_jumps.cpp

```diff
@@ -0,0 +1,44 @@
+//https://practice.geeksforgeeks.org/problems/minimum-number-of-jumps-1587115620/1
+#include <iostream>
+#include <algorithm>
+using namespace std;
+int minJumps(int arr[], int n)
```

Comment:
@vaibhav721  If possible include some logic in questions which can be used to understand the approach without looking the code :)

Reply:
Next time sure!